### ISSUE TITLE
[BUG] [1.8.9] Pump's screen is a mess

### ISSUE THREAD

**Issue by hron84** [K-4U/Hydraulicraft#215]
I don't want to be rude, but this is actually a fact: pumps own screen is an unreadable green mess. WAILA integration works, and tooltips are correct, but... those fancy screens!

![hydrapump-screenmess](https://cloud.githubusercontent.com/assets/88636/13514127/5e89bed0-e1a6-11e5-86d7-2b6328d6eac1.png)

From: HydCraft 1.8.9-3.1.65

**Comment by K-4U** [comment]
Heh, that is interesting.. Can you see anything in the logs about a font loader messing up?

**Comment by hron84** [comment]
I'll check it as soon as I can. Sadly today is a busy day.

**Comment by K-4U** [comment]
No worries, I don't know when I'll have the time to check it anyway.

**Comment by K-4U** [comment]
Any updates?

**Comment by hron84** [comment]
I could not reproduce the issue with the new modpack i started to play. The previous modpack I tried out had this issue, but it had other issues too, so I'm pretty sure it was an incompatibility problem from other mod that affected your displays somehow. I deleted the affected instance because it completelly wrecked my world made it unable to load anymore. Since I spent a whole lot of time into that world (and I had no backup because I never realized my favorite backup mod is already ported), I went furious and deleted every single bit of that modpack. As long as I remember, there was no font loading problem reported in the logs (I examined them very closely to fix the problem in my world). I close this issue, but I will reopen it as soon as I can reproduce the problem.
